install/helm/gloo/crds/gateway.gloo.solo.io_directresponses.yaml

```diff
@@ -13,6 +13,7 @@ spec:
   names:
     categories:
     - gloo-gateway
```

Comment:
do we want to remove `gloo-gateway` from existing crds so we don't have 3 different possible categories?

Reply:
Decided in standup to replace "k8sgateway" with "gloo-gateway". I added more context as a comment on the issue.